engines/standard_tasks/app/models/standard_tasks/register_decision_task.rb

```diff
@@ -110,6 +112,22 @@ def reject_letter
 
     private
 
+    def create_please_revise_card!
+      return unless revise_decision?
+
+      TaskFactory.build(Task, {
+        title: "Please Revise",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.